api/v3/Contact.php

```diff
@@ -376,6 +376,30 @@ function _civicrm_api3_contact_get_supportanomalies(&$params, &$options) {
     unset($params['filter.group_id']);
     $options['input_params']['group'] = $groups;
   }
+  if (isset($params['group'])) {
+    $groups = $params['group'];
+    $allGroups = CRM_Core_PseudoConstant::group();
+    if (is_array($groups) && in_array(key($groups), CRM_Core_DAO::acceptedSQLOperators(), TRUE)) {
+      $groupsArray = $groups[key($groups)];
+      foreach ($groupsArray as $group => &$ignore) {
```

Comment:
I don't like this stylistically - assigned unrequired vars - esp as a reference - I would use array_keys either when forming or when iterating the $groupsArray var